src/bin/pgcopydb/copydb.c

```diff
@@ -803,6 +791,11 @@ copydb_init_table_specs(CopyTableDataSpec *tableSpecs,
 		.indexSemaphore = &(specs->indexSemaphore)
 	};
 
+	if(tmpTableSpecs.source_pguri == NULL || tmpTableSpecs.target_pguri == NULL)
+	{
+		return false;
+	}
+
```

Comment:
Requires a log entry, using the constant ALLOCATION_FAILED_ERROR (so that we don't have to allocate memory for reporting OOM condition).